### PR TITLE
Add component guid mapping from excel

### PR DESCRIPTION
**Pull request by minhaj10p**
Added UUIDs against component checks in the excel sheet so that we don't end up generating new UUIDs everytime a new implementation code is generated. This PR addresses the mentioned integration.

### CODE REVIEW

File: impl/implementation.go [hunk 1]
```diff
@@ -29,25 +32,27 @@ type cdMap map[string]implementation.ComponentDefinition
 func GenerateImplementation(CSVS [][]string, p *profile.Profile, c Catalog) implementation.Implementation {
 
 	ComponentDefinitonMap := make(map[string]implementation.ComponentDefinition)
```
Comment (anweiss) [inline impl/implementation.go]: camel case

```suggestion
	componentDefinitonMap := make(map[string]implementation.ComponentDefinition)
```

Comment (minhaj10p) [reply impl/implementation.go]: resolved

**Comment by anweiss** [comment]
@minhaj10p once #37 is merged, you can go ahead and rebase this PR and we'll get it submitted.

File: impl/implementation.go [hunk 2]
```diff
@@ -28,38 +31,40 @@ type cdMap map[string]implementation.ComponentDefinition
 //GenerateImplementation generates implementation from component excel sheet
 func GenerateImplementation(CSVS [][]string, p *profile.Profile, c Catalog) implementation.Implementation {
 
-	ComponentDefinitonMap := make(map[string]implementation.ComponentDefinition)
-	checkAgainstGUUID := make(map[string]uuid.UUID)
+	componentDefinitonMap := make(map[string]implementation.ComponentDefinition)
+	checkAgainstGUID := make(map[string]uuid.UUID)
 
 	for i := RowIndex; i < TotalControlsInExcel; i++ {
```
Comment (anweiss) [inline impl/implementation.go]: camel case

```suggestion
	for i := rowIndex; i < totalControlsInExcel; i++ {
```

**Comment by anweiss** [comment]
@minhaj10p #37 has been merged. Please rebase master onto this PR and we'll get this merged as well.

**Comment by minhaj10p** [comment]
@anweiss I have rebased my master with upstream. Please go ahead and merge this PR. Thanks!

**Pull request merged**